microshift_release_notes/microshift-4-19-release-notes.adoc

```diff
@@ -70,6 +70,15 @@ With this release, {microshift-short} adds the Telemeter API. Lightweight attrib
 //[id="microshift-4-19-storage-docs-update_{context}"]
 //==== Updated content in the storage section
 
+[id="microshift-4-19-notable-technical-changes_{context}"]
+== Notable technical changes
+
+This release adds improvements related to the following components and concepts.
+
+[id="microshift-4-19-multus-standard-install_{context}"]
+=== Multus CNI now installed with {microshift-short}
+
+With this release, the {microshift-short} multiple networking plug-in is automatically installed. To deploy the Multus container network interface (CNI), you can either set the value in the {microshift-short} configuration file, or install the `microshift-multus` RPM and use it to create a configuration snippet. For more information, see xref:../microshift_networking/microshift_multiple_networks/microshift-cni-multus.adoc#microshift-cni-multus[About using multiple networks].
```

Comment:
I wouldn't say Multus is automatically installed. The manifests are now part of MicroShift, but they're not installed by default.

> install the `microshift-multus` RPM and use it to create a configuration snippet

RPM contains the configuration snippet/drop-in. No need to "use it to create".